### DISCUSSION TITLE
유니티 프로젝트 공유시 빌드잇에서 불러온 오브젝트의 쉐이더가 사라집니다

### DISCUSSION THREAD

**Discussion by IlsYoungOG** [naverz/zepeto-studio-kor#1186] [5. Q&A]
안녕하세요! 

현재 제작중인 월드가 
빌드잇에서 작업 후 -> 유니티로 불러와 오브젝트를 추가하였습니다. 

다른 팀원에게 프로젝트를 공유하기 위해 
아래 링크의 방법으로 프로젝트를 공유하였습니다. 
[https://angliss.cc/%EC%9C%A0%EB%8B%88%ED%8B%B0-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%EC%98%AE%EA%B8%B0%EB%8A%94-%EB%B2%95/](url)

그런데 공유받은 컴퓨터에서 프로젝트를 열어보니 
빌드잇에서 불러온 오브젝트들이 분홍색으로 뜨는 쉐이더가 사라진 현상이 나타났습니다. 
![빌드잇 분홍색](https://user-images.githubusercontent.com/107122738/185270343-8c548df6-7e73-4edc-b83c-c7fa688b43f7.png)

프로젝트 패키지 익스포트 할 시에 어떻게 공유를 해야 빌드잇 오브젝트 쉐이더가 포함되어 공유되는지 알고싶습니다! 

감사합니다.

**Comment by hwisulee** [comment]
![image](https://user-images.githubusercontent.com/62528282/185509746-bc43a79f-f246-44a2-8d09-f155dac14086.png)

유니티의 package manager - my registries에 빌드잇에서 사용하신 물체들의 테마가 설치되어 있는지 확인해보시겠어요?
빌드잇은 다음과 같은 유의 사항도 존재합니다!

![image](https://user-images.githubusercontent.com/62528282/185510217-e2c2f653-7cf6-4d78-9d1e-a8c69e3589f3.png)

**Reply by IlsYoungOG** [reply]
빌드잇 로더를 설치 하지 않았었더라구요 ㅠㅠ.. ㅋㅋ 해결 되었습니다! 알려주셔서 감사합니다!!!!!
